schemas/ophys/ophys_session.json

```diff
@@ -284,8 +170,8 @@
         "project_id",
         "session_type",
         "rig_id",
-        "probes",
+        "patch_cords",
         "lasers",
-        "ccf_version"
+        "coupling_array"
     ]
 }
```

Comment:
add empty line at end of file

Reply:
not sure what this means - just add an empty line? Is this something that needs to be there? I added one now - but let me know if I'm confused.